src/raw/mod.rs

```diff
@@ -145,27 +145,21 @@ fn h2(hash: u64) -> u8 {
 /// Proof that the probe will visit every group in the table:
 /// <https://fgiesen.wordpress.com/2015/02/22/triangular-numbers-mod-2n/>
 struct ProbeSeq {
-    bucket_mask: usize,
     pos: usize,
     stride: usize,
 }
 
-impl Iterator for ProbeSeq {
-    type Item = usize;
-
-    #[inline]
-    fn next(&mut self) -> Option<usize> {
+impl ProbeSeq {
+    fn move_next(&mut self, bucket_mask: usize) {
```

Comment:
ProbeSeq never changes it so it duplicates data already found in the table. Figured it was only there since `Iterator::next` couldn't take an argument.